iothub/device/tests/Authentication/DeviceClientConnectionStringExceptionTests.cs

```diff
@@ -120,22 +120,6 @@ public void DeviceClientConnectionStringEmptyConnectionStringExceptionTest()
             var deviceClient = IotHubDeviceClient.CreateFromConnectionString("");
         }
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentException))]
-        public void DeviceClientConnectionStringSASKeyX509CertExceptionTest()
```

Comment:
We only parse out the expected contents of a connection string. Any superfluous data in the connection string will be ignored